writeit/settings.py

```diff
@@ -376,23 +376,23 @@
     },
     # Resyncs popit every week
     'resync-popit-apis-every-week': {
-        'task': 'nuntium.tasks.pull_from_popolo_json',
+        'task': 'nuntium.tasks.update_all_instances',
         'kwargs': {
             'periodicity': '1W'
         },
         'schedule': crontab(hour=5, minute=30, day_of_week=1),
     },
     # Resyncs popit every day
     'resync-popit-apis-every-day': {
-        'task': 'nuntium.tasks.pull_from_popolo_json',
+        'task': 'nuntium.tasks.update_all_instances',
         'kwargs': {
             'periodicity': '1D'
         },
         'schedule': crontab(hour=5, minute=30),
     },
     # Resyncs popit twice every day
     'resync-popit-apis-twice-every-day': {
-        'task': 'nuntium.tasks.pull_from_popolo_json',
+        'task': 'nuntium.tasks.update_all_instances',
```

Comment:
Will this also cause the everypolitician source to sync?

If so, please look into deleting that - I tried but I'm not 100% sure that deleting the source links is ok - I think it is, as long as it doesn't delete the contact.

![Screenshot_2020-06-05_17-55-40](https://user-images.githubusercontent.com/235801/83897551-01f3c380-a756-11ea-9369-add30fe92666.png)

Reply:
Nope, it won't try to sync the everypolitician source. The task gets all the `WriteitInstancePopitInstanceRecord` instances which links instances to sources.

The everypolitician source isn't linked to any of the instances on production anymore.

![2020-06-09-16:13](https://user-images.githubusercontent.com/4767109/84158681-70919380-aa6c-11ea-9cc0-7d841b4092d6.png)